tests/instruments/test_all_instruments.py

```diff
@@ -156,6 +156,7 @@ def find_devices_in_module(module):
     "Keithley2400",
     "Keithley2450",
     "Keithley2600",
+    "Keithley2260B",
```

Comment:
How did the tests succeed previously without that line?

Reply:
I think because I added the deprecation on the `error` properties, following what has been done in the other files.

Reply:
Adding an instrument to that list, removes the checks for all properties.

The idea is, to eventually remove all instruments from the list.

Therefore, it is better to fix the code (docstring) to pass the test.

Reply:
Also other files does not have documentation in `error`. I was confused about this. So, this file was the only one with check enabled while the other had the check disabled.

Reply:
Changes done